dev/MRTCore/mrt/Microsoft.ApplicationModel.Resources/UnpackagedTests/TestClass.cs

```diff
@@ -107,47 +107,89 @@ public class TestClass
     {
         private ActivationContext m_context = new ActivationContext();
         private static string m_assemblyFolder = Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location);
+        private static string m_exeFolder = Path.GetDirectoryName(System.Diagnostics.Process.GetCurrentProcess().MainModule.FileName);
+        private static bool m_rs5 = false;
+
+        private static void Cleanup()
+        {
+            GC.Collect(); // Force any ResourceManager objects to be cleaned up.
+
+            if (File.Exists(Path.Combine(m_exeFolder, "resources.pri")))
+            {
+                File.Delete(Path.Combine(m_exeFolder, "resources.pri"));
+            }
+            if (File.Exists(Path.Combine(m_exeFolder, "te.processhost.pri")))
+            {
+                File.Delete(Path.Combine(m_exeFolder, "te.processhost.pri"));
+            }
+            if (m_exeFolder != m_assemblyFolder)
+            {
+                if (File.Exists(Path.Combine(m_exeFolder, "resources.pri.standalone")))
+                {
+                    File.Delete(Path.Combine(m_exeFolder, "resources.pri.standalone"));
+                }
+            }
+        }
 
         [AssemblyInitialize]
         public static void ModuleSetup(TestContext testContext)
         {
             // Clean up any left over files just in case
-            File.Delete(Path.Combine(m_assemblyFolder, "resources.pri"));
-            File.Delete(Path.Combine(m_assemblyFolder, "te.processhost.pri"));
+            File.Delete(Path.Combine(m_exeFolder, "resources.pri"));
+            File.Delete(Path.Combine(m_exeFolder, "te.processhost.pri"));
+            if (m_exeFolder != m_assemblyFolder)
+            {
+                File.Delete(Path.Combine(m_exeFolder, "resources.pri.standalone"));
+            }
+            m_rs5 = (System.Environment.OSVersion.Version.Build < 18362);
         }
 
-        [TestCleanup]
-        public static void TestCleanup()
+        [AssemblyCleanup]
+        public static void ModuleCleanup()
         {
-            GC.Collect(); // Force any ResourceManager objects to be cleaned up.
+            Cleanup();
+        }
 
-            if (File.Exists(Path.Combine(m_assemblyFolder, "resources.pri")))
-            {
-                File.Delete(Path.Combine(m_assemblyFolder, "resources.pri"));
-            }
-            if (File.Exists(Path.Combine(m_assemblyFolder, "te.processhost.pri")))
-            {
-                File.Delete(Path.Combine(m_assemblyFolder, "te.processhost.pri"));
-            }            
+        [TestInitialize]
+        public static void TestSetup()
+        {
+            Cleanup();
         }
 
-        [TestMethod]
-        public void DefaultResourceManager()
+        [TestCleanup]
+        public static void TestCleanup()
+        { 
+            // There is random ACCESS_DENIED error during cleanup running in Helix.
```

Comment:
nice job finding a way to get this to work without having to use isolation on each method. :)

Reply:
actually isolation didn't work on Helix. It's more stubborn than what you had encountered.